autogl/module/train/__init__.py

```diff
@@ -26,6 +26,7 @@ def register_trainer_cls(cls):
 
 from .graph_classification_full import GraphClassificationFullTrainer
 from .node_classification_full import NodeClassificationFullTrainer
+from .node_classification_het import NodeClassificationHetTrainer
```

Comment:
You need to put NodeClassificationHetTrainer keyword in the `__all__` list in this file.